Makefile

```diff
@@ -88,7 +89,7 @@ test: pre-test
 run-integration-tests: export DOCKER_HOSTNAME?=localhost:5000
 run-integration-tests: export DOCKER_NAMESPACE?=fybrik-system
 run-integration-tests: export VALUES_FILE=charts/fybrik/integration-tests.values.yaml
-run-integration-tests: export HELM_SETTINGS=--set "manager.solver.enabled=true"
+run-integration-tests: export HELM_SETTINGS=--set "manager.solver.enabled=false "
```

Comment:
Please leave "true". We run integration tests with solver enabled. 
If solver is not integrated (the corresponding image is empty), the enabled flag won't matter.

Reply:
I cannot run solver on my laptop, so I set it to false, and mistakenly committed it. Thanks